packages/cozy-procedures/src/components/PersonalDataForm.jsx

```diff
@@ -32,6 +33,7 @@ class PersonalDataForm extends React.Component {
       <div>
         <Topbar title={t('personalDataForm.title')} />
         <SubTitle>{t('personalDataForm.subtitle')}</SubTitle>
+        <FormFillingStatus />
```

Comment:
We're missing the props here.

Reply:
the props are given by the container

Reply:
Oh right. In that case the problem is somewhere else, because the counter appears even when no fields are completed at the moment :/